dapr-spring/dapr-spring-boot-tests/src/main/java/io/dapr/spring/boot/testcontainers/service/connection/DaprContainerConnectionDetailsFactory.java

```diff
@@ -0,0 +1,45 @@
+package io.dapr.spring.boot.testcontainers.service.connection;
+
+import io.dapr.spring.boot.autoconfigure.client.DaprConnectionDetails;
+import io.dapr.testcontainers.DaprContainer;
+import org.springframework.boot.testcontainers.service.connection.ContainerConnectionDetailsFactory;
+import org.springframework.boot.testcontainers.service.connection.ContainerConnectionSource;
+
```

Comment:
We could rename this to `DaprConnectionDetailsFactory` so it is similar to other connection details factory like Redis, Mongo, Kafka, etc

Reply:
@artur-ciocanu I am following testcontainers conventions as the connectiondetails for this will be extracted from the DaprContainer